### ISSUE TITLE
Cannot type "." in a float column

### ISSUE THREAD

**Issue by OliverEvans96** [prisma/studio#571]
1. Prisma version (`prisma -v` or `npx prisma -v`):

```
% yarn prisma -v
yarn run v1.22.10
$ /home/oliver/farmly/plant-data/node_modules/.bin/prisma -v
@prisma/cli          : 2.11.0-dev.9
@prisma/client       : 2.10.1
Current platform     : debian-openssl-1.1.x
Query Engine         : query-engine 2034ce0e038ece25b48b5180b41ccb48d2c3344f (at node_modules/@prisma/cli/query-engine-debian-openssl-1.1.x)
Migration Engine     : migration-engine-cli 2034ce0e038ece25b48b5180b41ccb48d2c3344f (at node_modules/@prisma/cli/migration-engine-debian-openssl-1.1.x)
Introspection Engine : introspection-core 2034ce0e038ece25b48b5180b41ccb48d2c3344f (at node_modules/@prisma/cli/introspection-engine-debian-openssl-1.1.x)
Format Binary        : prisma-fmt 2034ce0e038ece25b48b5180b41ccb48d2c3344f (at node_modules/@prisma/cli/prisma-fmt-debian-openssl-1.1.x)
Studio               : 0.304.0
```

2. Logs from Developer Tools Console or Command line, if any:
N/A

3. Does the issue persist even after updating to the latest `prisma` alpha? (`npm i -D @prisma/cli@dev`)
Yes

4. Prisma schema (if relevant):
N/A

I can only type numbers in a float column, but I can't type a decimal point ("."). I can, however, *paste* a decimal point, unlike other special characters.

It seems like this should be a quick fix, and would be quite helpful!

Thanks!
Oliver

**Comment by madebysid** [comment]
Hey @OliverEvans96, thanks for reporting this! I can indeed reproduce this. I'll let you know when I push out a fix!

**Comment by madebysid** [comment]
@OliverEvans96 This should now be fixed with `@prisma/cli` version `2.11.0-dev.20`, could you try updating and confirm?

As before, I'd recommend switching to a stable version (2.11.0) when it comes out next week so you're not caught out by accidental breaking changes :)
